app/controllers/api/braintree_controller.rb

```diff
@@ -37,11 +38,15 @@ def subscription_options
     {
       amount: params[:amount].to_f,
       plan_id: '35wm',
-      payment_method_token: Payment.customer(params[:email]).default_payment_method.token
+      payment_method_token: default_payment_method_token
     }
   end
 
   def braintree
     PaymentProcessor::Clients::Braintree
   end
+
+  def default_payment_method_token
+    @token ||= ::Payment.customer(params[:email]).try(:card_vault_token)
```

Comment:
`card_vault_token` is payment method token stored in the vault:
http://www.imdb.com/title/tt1556190/ It's passed as `token` when a transaction is made. I should have named the attribute `payment_method_token`!